tests/shared/src/test/resources/resolutions/org.webjars.bower/malihu-custom-scrollbar-plugin/3.1.5

```diff
@@ -1,3 +1,3 @@
-org.webjars.bower:jquery:3.1.1:compile
```

Comment:
Yeah, the [corresponding test](https://github.com/coursier/coursier/blob/14f297b4f7ad8cf83abfe030825cd5b51bee64e3/tests/shared/src/test/scala/coursier/test/CentralTests.scala#L279) is kind of flaky, as it checks some version interval, but the corresponding dependency still gets updated from time to time. I guess the corresponding metadata should be kept as resources in this repo, so that the test doesn't need to be udpated this way.